datastore/data-loader/client_knmi_station_ingest.py

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/env python3
```

Comment:
I guess comparing this with `client_knmi_station.py` making this work with one file would only make it more confusing.

But can we move some shared helper stuff to a shared file. Like everything above `netcdf_file_to_requests()` and also `generate_parameter_name()` (the only real difference seems to be the level, which can easily be translated after calling this function?

Reply:
Moved everything to `utilities.py` in https://github.com/EURODEO/e-soh/pull/209/commits/c5de77e91087ab4e7d9c9af0c438c406db00e4bc and doing some translations where needed in the `data-loaders`